reference/chromecast/ADBMobile.media.html

```diff
@@ -1582,7 +1582,7 @@ <h4 class="name" id=".setDelegate"><span class="type-signature">(static) </span>
     <p>Sets an instance of MediaDelegate to be used by the ADBMobile library.<br>
 The delegate object passed should have 2 functions.</p>
 <ol>
-<li><code>getCurrentPlaybackTime()</code> - Returns current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item. For live streaming, the value is specified as the number of seconds since midnight UTC on that day.</li>
+<li><code>getCurrentPlaybackTime()</code> - Returns the current playhead in seconds. For video-on-demand (VOD), the value is specified in seconds from the beginning of the media item.  For live streaming, if the player does not provide information about the content duration, the value can be specified as the number of seconds since midnight UTC of that day.</li>

```

Comment:
nit: there is an extra space before _**For** Live..._

Reply:
removed the extra space.